_docs_integrate/integration-example.md

```diff
@@ -212,16 +212,16 @@ All three paths should result in a screen similar to the one below, where you ca
 
 !["Add contact" screen]( {{ '/assets/images/add-contact-screen.jpg' | relative_url }} )
 
-Finally, fill out the required fields and click on "Add contact" to send the Relationship Request. This will create a new Relationship between the App and the Connector. This Relationship has the status `Pending` for now.
+Finally, fill out the required fields and click on "Add contact" to send the Relationship. This will create a new Relationship between the App and the Connector. This Relationship has the status `Pending` for now.
```

Comment:
`"Pending"`

Reply:
I don't understand this one

Reply:
This Relationship has the status `"Pending"` for now. 
only adding "" because it's a value.

Reply:
I like it more without the "s here. `s are used for highliting important information here, but in this text we don't talk about a string that is used, we talk about a specific status.

Reply:
IMO it is not a matter of taste. Please try to be consistent as possible and stick to `"Pending"` here, as we do in the data model overview (https://enmeshed.eu/integrate/data-model-overview#relationship) and other scenario documentation pages (for example, see https://enmeshed.eu/integrate/establish-relationships).

Reply:
Delaying this since it will be a part of a more general discussion